stock_quant_manual_assign/wizard/assign_manual_quants.py

```diff
@@ -42,18 +46,20 @@ def assign_quants(self):
         move = self.env['stock.move'].browse(self.env.context['active_id'])
         move.picking_id.mapped('pack_operation_ids').unlink()
         quants = []
+        # Mark as recompute pack needed
+        if move.picking_id:  # pragma: no cover
+            move.picking_id.recompute_pack_op = True
         for quant_id in move.reserved_quant_ids.ids:
             move.write({'reserved_quant_ids': [[3, quant_id]]})
         for line in self.quants_lines:
```

Comment:
that line of code is from v09, so in case we rewrite it, shall we need to update for the v09 too?